fix links (#23)
apiobuild/apiobuild.github.io: content/blog/collect-email-addresses-without-email-marketing-tools.md

Remembering the time when you have to download the csv file from your email marketing provider, then struggle with matching the format to another digital tool.

> With **<a href="https://telescope.apiobuild.com/app/waitress" target="_blank">Waitress</a>**, the freedom to manage your own data makes it easier to integrate with any third-party software. **Data is available through API.** It's easily customizable and integrable with other third-party marketing solutions. You can easily connect the subscriber data to ticketing platform, Google Analytics, and more!

Author: ```suggestion
With **<a href="https://telescope.apiobuild.com/app/waitress" target="_blank">Waitress</a>**, you have the freedom to manage your own data makes it easier to integrate with any third-party software. **Data is available through API.** It's easily customizable and integrable with other third-party marketing solutions. You can easily connect the subscriber data to ticketing platform, Google Analytics, and more!
```

> Visit **<a href="https://telescope.apiobuild.com/" target="_blank">Telescope</a>** to view our service offering portfolio .

Author: ```suggestion
Visit **<a href="https://telescope.apiobuild.com/" target="_blank">Telescope</a>** to check out more apio's microservices offerings.
```
